state_management/javascript/http/order-processor/index.js

```diff
@@ -1,42 +1,40 @@
-import axios from "axios";
+import axios from "axios"
 
-const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
-const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
-const DAPR_STATE_STORE = 'statestore'
+const DAPR_HOST = process.env.DAPR_HOST ?? "http://localhost"
```

Comment:
split protocol from host, use protocol in constructing the url

Reply:
I did split the port from host although I'm not sure if it is done how you expected. I don't see any env vars which indicate protocol.  Because of this, there is complexity introduced using protocol to determine if PORT should be assigned DAPR_HTTP_PORT or DAPR_GRPC_PORT. This could intimidate new users as they would likely choose one protocol for their service and it would be very rare to write an service to support BOTH.

I didn't see any environment variable set which indicated which protocol was intended to be used.
These are the only vars I knew about:
```
DAPR_GRPC_PORT
DAPR_HTTP_PORT
DAPR_METRICS_PORT
```
Is there a var given to the application that specifies protocol?

In my opinion, this request for gRPC is a not appropriate. This PR was about making the samples consistent and runnable. The original tutorials included the protocol with the host.
This is like a new feature to make these samples support gRPC.